services/workflows-service/.prettierignore

```diff
@@ -2,4 +2,5 @@ node_modules/
 dist/
 prisma/migrations/
 package-lock.json
-coverage/
\ No newline at end of file
+coverage/
+src/ui-definition/utils
```

Comment:
Let's make sure utils are formatted. Otherwise each IDE might output different format and noisy PRs.